src/graphql/config.ts

```diff
@@ -7,7 +7,7 @@ export const getGraphQlConfig = (
   const isProduction = configService.get<string>('NODE_ENV') === 'production';
   return {
     playground: true,
-    // Generate graphql schema for dev to store it repo
+    // Generate graphql schema for dev to store it in repo
     autoSchemaFile: isProduction ? true : 'src/graphql/schema.gql',
```

Comment:
This is nothing to do with performance :) Basically i though that it would be great to save the schema in the repo, so we or other members of the team would easily check it out :) and of course we'll be able to easier track the changes